cmd/cli/plugin/package/go.mod

```diff
@@ -12,6 +12,8 @@ replace (
 	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ../../../../cli/runtime
 	github.com/vmware-tanzu/tanzu-framework/packageclients => ../../../../packageclients
 	github.com/vmware-tanzu/tanzu-framework/tkg => ../../../../tkg
+	github.com/vmware-tanzu/tanzu-framework/tkr => ../../../../tkr
+	github.com/vmware-tanzu/tanzu-framework/util => ../../../../util
```

Comment:
Why these are needed for package plugin and would all the changes in this go mod be needed for [secret plugin go.mod](https://github.com/vmware-tanzu/tanzu-framework/blob/main/cmd/cli/plugin/secret/go.mod) as well?

Reply:
This is because cmd/cli/plugin/package uses `tkg` which uses both `tkr` and `util`. Unlike [secret plugin](https://github.com/vmware-tanzu/tanzu-framework/blob/main/cmd/cli/plugin/secret/go.mod) that doesn't actually use `tkg`.